frontend/cypress/perf/common.ts

```diff
@@ -39,15 +50,23 @@ const measureLoadTime = (name: string, visits: number, loadUrl: string, loadElem
     .then(() => {
       sum = sum / visitsArray.length;
 
-      const contents = `${name} load time: ${(sum / 1000).toPrecision(5)} seconds\n`;
+      const contents = `${name} load time: ${compareToBaseline(sum, baseline)}\n`;
       cy.writeFile(reportFilePath, contents, { flag: 'a+' });
     });
-}
+};
+
+export const measureListsLoadTime = (name: string, baseline: number, listUrl: string): void => {
+  measureLoadTime(name, baseline, listUrl, '.pf-v5-c-toolbar');
+};
 
-export const measureListsLoadTime = (name: string, visits: number, listUrl: string): void => {
-  measureLoadTime(name, visits, listUrl, '.pf-v5-c-toolbar');
-}
+export const measureDetailsLoadTime = (name: string, baseline: number, detailsUrl: string): void => {
+  measureLoadTime(name, baseline, detailsUrl, '.pf-v5-c-tabs');
+};
 
-export const measureDetailsLoadTime = (name: string, visits: number, detailsUrl: string): void => {
-  measureLoadTime(name, visits, detailsUrl, '.pf-v5-c-tabs');
-}
+export const compareToBaseline = (resultMS: number, baseline: number): string => {
+  // to seconds
+  const result = resultMS / 1000;
+  const resultSeconds = result.toPrecision(5);
+  const difference = baseline - result;
+  return `${resultSeconds} sec, baseline: ${baseline} sec, difference: ${difference.toPrecision(5)} sec\n`;
```

Comment:
nit: it would be nice to have `+` or `-` in the difference like:
`App Outbound Metrics load time: 0.45238 sec, baseline: 1 sec, difference: -0.54762 sec`
`App Outbound Metrics load time: 2 sec, baseline: 1 sec, difference: +1 sec`

Reply:
yeah good point,